Tests/SwiftSyntaxBuilderTest/VariableTests.swift

```diff
@@ -371,4 +371,20 @@ final class VariableTests: XCTestCase {
       assertBuildResult(builder, expected, line: line)
     }
   }
+
+  // https://github.com/apple/swift-syntax/issues/1815
+  func testSpacingInDictionaryType() throws {
+    let header = PartialSyntaxNodeString(stringLiteral: "static var mirror: Dictionary<String, Any.Type>")
+    let buildable = try VariableDeclSyntax(header) {
+
+    }
```

Comment:
Do you need to explicitly construct a `PartialSyntaxNodeString` here? Wouldn’t any of the following do the job equally well here? Also: Shouldn’t this test case be in *SwiftBasicFormatTest* instead of *SwiftSyntaxBuilderTest*?

Wouldn’t 

```swift
let buildable = """
static var mirror: Dictionary<String, Any.Type> {}
"""
```

or 

```swift
let buildable = try VariableDeclSyntax("static var mirror: Dictionary<String, Any.Type>") {}
```

Reply:
It was to come as close ro the GitHub issue as possible. 
I can change to the last example if you want.

Reply:
Ah, I see. In that case either really makes sense. Feel free to keep it.

Reply:
Changed to the last example